docs/release/release-tasks.md

```diff
@@ -318,6 +318,15 @@ The goal of this task to make the book for the current release available under e
 3. Update references in introduction.md only on the main branch (drop unsupported versions, add the new release version).
    <br>Prior art: [Add release 1.2 book link](https://github.com/kubernetes-sigs/cluster-api/pull/6697)
 
+#### Generate weekly PR updates to post in Slack
+The goal of this task is to keep the CAPI community updated on recent PRs that have been merged. This is done by using the weekly update tool in `hack/tools/release/weekly/main.go`. Here is how to use it:
+1. Checkout the latest commit on the release branch, e.g. `release-1.4`, or the main branch if the release branch doesn't yet exist (e.g. beta release).
+2. Generate the weekly update with the following command:
+   ```bash
+   go run hack/tools/release/weekly/main.go notes --since YYYY-MM-DD --until YYYY-MM-DD --milestone v1.x
```

Comment:
We could have a new Makefile target for it as `release-notes-tool`, i.e: 
`release-weekly-update` and use that here